danielMunteanu/src/main/java/Order.java

```diff
@@ -0,0 +1,67 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+class Order {
+    public long id;
+    int counter;
+    public String date;
```

Comment:
The date format should of another type. Make sure you are on the correct way by reading the task one more time.

Reply:
fixed in next commit